### PR TITLE
improve the channel map filtering

### PR DESCRIPTION
**Pull request by moonyuet**
## Changelog Description
The channel map export in the latest develop cannot filter some maps when the maps include similar names (i.e. Color and EmissionColor). This PR is to make sure the current channel map filtering would filter the maps as expected. 
image

## Additional info
n/a

## Testing notes:

1. Launch Substance Painter with your project
2. Create Texture Set
3. Use RsMaterialTemplate and add base color. Following the setting as the screenshots showed. "Channel Type should be `_Color` for BaseColor if you are using RsMaterialTemplate"
image
image

4. Publish
5. It should only publish Color Map.

### CODE REVIEW

File: server/settings/creator_plugins.py [hunk 1]
```diff
@@ -22,38 +22,38 @@ class CreatorsModel(BaseSettingsModel):
...
+            {"name": "Blending Mask", "value": "_BlendingMask"},
+            {"name": "Translucency", "value": "_Translucency"},
+            {"name": "Scattering", "value": "_Scattering"},
+            {"name": "Scatter Color", "value": "_ScatterColor"},
+            {"name": "Sheen Opacity", "value": "_SheenOpacity"},
+            {"name": "Sheen Color", "value": "_SheenColor"},
+            {"name": "Coat Opacity", "value": "_CoatOpacity"},
+            {"name": "Coat Color", "value": "_CoatColor"},
+            {"name": "Coat Roughness", "value": "_CoatRoughness"},
+            {"name": "Coat Specula rLevel",
```
Comment (BigRoy) [inline server/settings/creator_plugins.py:54]:
```suggestion
            {"name": "Coat Specular Level",
```

File: server/settings/creator_plugins.py [hunk 2]
```diff
@@ -22,38 +22,38 @@ class CreatorsModel(BaseSettingsModel):
...
-            {"name": "Glossiness", "value": "Glossiness"},
+             "value": "_SpecularEdgeColor"},
+            {"name": "Opacity", "value": "_Opacity"},
+            {"name": "Displacement", "value": "_Displacement"},
+            {"name": "Glossiness", "value": "_Glossiness"},
             {"name": "Anisotropy Level",
-             "value": "Anisotropylevel"},
-            {"name": "Ambient Occulsion", "value": "AO"},
+             "value": "_Anisotropylevel"},
+            {"name": "Ambient Occulsion", "value": "_AO"},
```
Comment (BigRoy) [inline server/settings/creator_plugins.py:37]:
```suggestion
            {"name": "Ambient Occlusion", "value": "_AO"},
```

File: server/settings/creator_plugins.py [hunk 3]
```diff
@@ -22,38 +22,38 @@ class CreatorsModel(BaseSettingsModel):
...
-            {"name": "Scatter Color", "value": "ScatterColor"},
-            {"name": "Sheen Opacity", "value": "SheenOpacity"},
-            {"name": "Sheen Color", "value": "SheenColor"},
-            {"name": "Coat Opacity", "value": "CoatOpacity"},
-            {"name": "Coat Color", "value": "CoatColor"},
-            {"name": "Coat Roughness", "value": "CoatRoughness"},
-            {"name": "CoatSpecularLevel",
-             "value": "Coat Specular Level"},
-            {"name": "CoatNormal", "value": "Coat Normal"}
+             "value": "_Anisotropyangle"},
```
Comment (BigRoy) [inline server/settings/creator_plugins.py:39]: Should we move this one up so it's next to Anisotropy Level?

**Review by BigRoy — APPROVED** [review]
Looks good - did have some cosmetic notes. Couldn't test myself.

Might need a test-run by @LiborBatek ?

**Review by LiborBatek — APPROVED** [review]
I have followed testing steps and all working fine...

I was able to produce publish for ```rsMaterial``` and ```Color``` channel only without any errors shown.

Screenshot 2024-07-30 094517

**Pull request merged**